onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -515,18 +515,30 @@ static Status HandleAutoPad(const Shape& input_shape,
 static Status GetBinaryOpQuantizationScaleAndZeroPoint(
     const ModelBuilder& model_builder, const Node& node,
     float& a_scale, float& b_scale, float& y_scale,
-    int32_t& a_zero_point, int32_t& b_zero_point, int32_t& y_zero_point) ORT_MUST_USE_RESULT;
+    int32_t& a_zero_point, int32_t& b_zero_point, int32_t& y_zero_point, std::unique_ptr<ConstNodesToOptimize>& qdq_node_group) ORT_MUST_USE_RESULT;
```

Comment:
>qdq_node_group [](http://example.com/codeflow?start=112&length=14)

general comment: passing unique_ptr& is rarely required (e.g., for modifying the ownership of the unique_ptr). here, it looks like const T* would work. if no ownership semantics are needed for a parameter, consider using raw pointers or references directly

Reply:
Yes. Scott also mentioned this. Will change them all to raw pointers.